src/Symfony/Component/Amqp/BrokerInterface.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Amqp;
+
+// WIP
+
+interface BrokerInterface
```

Comment:
It's a looot of methods. Can't we break it into multiple interfaces? (if there is any point though...)

Reply:
Do we even need an interface?